integrations/access/jira/config.go

```diff
@@ -48,33 +63,47 @@ func (c *Config) CheckAndSetDefaults() error {
 	if err := c.Teleport.CheckAndSetDefaults(); err != nil {
 		return trace.Wrap(err)
 	}
-	if c.Jira.URL == "" {
-		return trace.BadParameter("missing required value jira.url")
+
+	if err := c.Jira.CheckAndSetDefaults(); err != nil {
+		return trace.Wrap(err)
 	}
-	if !strings.HasPrefix(c.Jira.URL, "https://") {
-		return trace.BadParameter("parameter jira.url must start with \"https://\"")
+
+	if c.EnableWebhook {
```

Comment:
So now that you need to enable webhook explicitly, is this backwards compatible with self-hosted version of the plugin? We don't want to have self-hosted change their configuration. Does this default to "true" in self-hosted scenario or should we invert this to `DisableWebhook`?

Reply:
Done